### PR TITLE
docs: Standardize component descriptions

### PR DESCRIPTION
**Pull request by joebuono**
#### Description of changes

This PR standardizes the component descriptions. 

#### Description of how you validated changes

Ran the docs locally. 

#### Checklist

- [X] PR description included
- [X] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/src/pages/[platform]/components/heading/index.page.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Heading
-description: A Heading primitive is used for semantic section heading text representing six levels.
+description: Headings render HTML Section Heading elements <h1>–<h6>.
```
Comment (zchenwei) [inline docs/src/pages/[platform]/components/heading/index.page.mdx:3]: nit: maybe just HTML Heading？

File: docs/src/pages/[platform]/components/icon/index.page.mdx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Icon
-description: The icon component displays simple vector graphics for use in other components like Buttons.
+description: Icons are used to display scalable vector graphics (SVG’s).
```
Comment (zchenwei) [inline docs/src/pages/[platform]/components/icon/index.page.mdx:3]: nit: Scalable Vector Graphics (SVG)?

File: docs/src/pages/[platform]/components/link/index.page.mdx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Link
-description: Links are customizable and themable elements used for Navigation. By default Links render an anchor tag but can be configured to be used with routing libraries.
+description: Links help users navigate within an application. Links render an anchor element <a> by default, and can also be used with routing libraries.
```
Comment (zchenwei) [inline docs/src/pages/[platform]/components/link/index.page.mdx:3]: For a brief description, I think we can make it simple - `Links semantically renders an <a> and are primarily used for navigation.` Thoughts?

**Review by zchenwei — APPROVED** [review]
LGTM! Left a few thoughts

File: docs/src/pages/[platform]/components/flex/index.page.mdx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Flex
-description: A layout container using Flexbox.
+description: A layout container using CSS Flexbox.
```
Comment (reesscot) [inline docs/src/pages/[platform]/components/flex/index.page.mdx:3]: This doesn't seem to follow the same pattern?

File: docs/src/pages/[platform]/components/divider/index.page.mdx [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Divider
-description: A Divider creates separations in content. Dividers can help organize content and establish visual rhythm.
+description: Dividers help organize content, separate sections and establish visual rhythm.
```
Comment (reesscot) [inline docs/src/pages/[platform]/components/divider/index.page.mdx:3]: Please use a singular primitive name `Divider`

File: docs/src/pages/[platform]/components/grid/index.page.mdx [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Grid
-description: Grid provides a CSS grid container.
+description: A layout container using CSS Grid.
```
Comment (reesscot) [inline docs/src/pages/[platform]/components/grid/index.page.mdx:3]: Need to follow same pattern.

File: docs/src/pages/[platform]/components/heading/index.page.mdx (hunk 1 above)
Comment (reesscot) [reply docs/src/pages/[platform]/components/heading/index.page.mdx:3]: +1 Can you adapt the previous version calling out the semantic meaning to the new format?
```
Heading renders semantic section heading text representing six levels
```

**Review by reesscot — APPROVED** [review]
LGTM, let's just change to singular for primitive names

**Pull request merged**